irods/templates/sftp/etc/sftpgo/sftpgo.json.j2

```diff
@@ -9,8 +9,8 @@
     },
     "setstat_mode": 0,
     "temp_path": "",
-    "proxy_protocol": 0,
-    "proxy_allowed": [],
+    "proxy_protocol": 1,
+    "proxy_allowed": {{ groups['proxy'] | map('extract', hostvars, ['ansible_default_ipv4', 'address']) }},
```

Comment:
Instead of coupling the playbook to the existence of a group, use a variable like `sftp_proxy_allowed` that would hold a list of allowed servers. Then, in the environment specific deployments, couple this variable to the `proxy` group, i.e., `sftp_proxy_allowed = groups['proxy']`.  This allows for the Data Store to be deployed with SFTP without requiring proxy server, which some of our customers don't have.

Reply:
Added a new variable `sftp_proxy_allowed`.

Reply:
About the unit test error, I don't see it in my environment.

```
TASK [test sftpgo.json contents] **************************************************************************************************************************************************
ok: [dstesting-sftp-1.dstesting_default] => {
    "changed": false,
    "msg": "All assertions passed"
}
```